### PR TITLE
Fix conditions index

### PR DESCRIPTION
**Pull request by amhursh**
Fixed conditions db issue with mean_vis being mislabeled. Added labels and units to the data showing on the conditions index and conditions show views. For db issue the re-seeding process was necessary.

### CODE REVIEW

File: app/views/conditions/index.erb [hunk 1]
```diff
@@ -1,19 +1,21 @@
 <h1>Weather Conditions</h1>
 
 <% @conditions.each do |condition| %>
-  <a href="conditions/<%= condition.id %>"><%= condition.date %></a>
+  <h3>
+    <a href="conditions/<%= condition.id %>"><%= condition.date %></a>
+  </h3>
```
Comment (mimizotti) [inline app/views/conditions/index.erb:7]: Is this tag consistent across the website?

File: app/views/conditions/index.erb [hunk 2]
```diff
@@ -1,19 +1,21 @@
...
 
-  <%= condition.max_temperature %>
+  <p>Max Temp: <%= condition.max_temperature %> °F</p>
```
Comment (mimizotti) [inline app/views/conditions/index.erb:9]: Looks good.

File: app/views/conditions/index.erb [hunk 3]
```diff
@@ -1,19 +1,21 @@
...
 
-  <%= condition.min_temperature %>
+  <p>Min Temp: <%= condition.min_temperature %> °F</p>
```
Comment (mimizotti) [inline app/views/conditions/index.erb:11]: Looks good.

File: app/views/conditions/index.erb [hunk 4]
```diff
@@ -1,19 +1,21 @@
...
 
-  <%= condition.mean_humidity %>
+  <p>Mean Humidity: <%= condition.mean_humidity %> %</p>
```
Comment (mimizotti) [inline app/views/conditions/index.erb:13]: Looks good.

File: app/views/conditions/index.erb [hunk 5]
```diff
@@ -1,19 +1,21 @@
...
 
-  <%= condition.mean_visibility %>
+  <p>Mean Visibility: <%= condition.mean_visibility %> miles</p>
```
Comment (mimizotti) [inline app/views/conditions/index.erb:15]: Looks good.

File: app/views/conditions/index.erb [hunk 6]
```diff
@@ -1,19 +1,21 @@
...
 
-  <%= condition.mean_windspeed %>
+  <p>Mean Windspeed: <%= condition.mean_windspeed %> mph</p>
```
Comment (mimizotti) [inline app/views/conditions/index.erb:17]: Looks good.

File: app/views/conditions/index.erb [hunk 7]
```diff
@@ -1,19 +1,21 @@
...
 
-  <%= condition.precipitation %>
+  <p>Precipitation: <%= condition.precipitation %> %</p>
```
Comment (mimizotti) [inline app/views/conditions/index.erb:19]: Looks good.

File: app/views/conditions/show.erb [hunk 8]
```diff
@@ -1,14 +1,15 @@
-<%= @condition.date %>
+  <h1>
+    <%= @condition.date %>
+  </h1>
```
Comment (mimizotti) [inline app/views/conditions/show.erb:4]: Is this tag consistent across the website (specifically across all dashboards?)

File: db/migrate/20170823053830_change_conditions.rb [hunk 9]
```diff
@@ -22,7 +22,7 @@ def change
       t.rename :max_temperature_f, :max_temperature
       t.rename :mean_temperature_f, :mean_temperature
       t.rename :min_temperature_f, :min_temperature
-      t.rename :mean_visibility_miles, :min_visibility
+      t.rename :mean_visibility_miles, :mean_visibility
```
Comment (mimizotti) [inline db/migrate/20170823053830_change_conditions.rb:25]: Nice catch!

**Review by mimizotti — APPROVED** [review]
Changes look good!

File: app/views/conditions/index.erb (hunk 1 above)
Comment (amhursh) [reply app/views/conditions/index.erb:7]: I think not. I've added tag standardization to the waffle issue list.

File: app/views/conditions/show.erb (hunk 8 above)
Comment (amhursh) [reply app/views/conditions/show.erb:4]: see above

**Pull request merged**